crates/solidity/inputs/language/bindings/rules.msgb

```diff
@@ -2060,15 +2060,48 @@ inherit .star_extension
   edge @enum.def -> type
   edge type -> type_enum_type
   edge type_enum_type -> @enum.lexical_scope
-}
 
-@enum [EnumDefinition
+  ; Path to link min and max to the enum's scope (#1158)
+  ; It resolves paths of the form `type(<enum>).min.<something>`
+  node typeof
+  attr (typeof) pop_symbol = "@typeof"
+
+  node built_in_member
+  attr (built_in_member) pop_symbol = "."
+  node min_built_in
+  attr (min_built_in) pop_symbol = "min"
+  node max_built_in
+  attr (max_built_in) pop_symbol = "max"
```

Comment:
Both `min` and `max` only exist in Solidity >= 0.6.8. It's not critical, but to be consistent with the built-in definitions these should be only added for those versions.